roles/ceilometer-data/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: install ceilometer compute service
+  upstart_service: name=ceilometer-agent-compute
+                   user=ceilometer
+                   cmd=/usr/local/bin/ceilometer-agent-compute
+  notify:
+    - restart ceilometer services
+
+- name: pip install libvirt-python in venv on compute
+  pip: name=libvirt-python virtualenv="{{ ceilometer.virtualenv }}"
+  notify:
+    - restart ceilometer services
```

Comment:
I am fine with moving libvirt-python to install on all nodes, but Niraj originally advised me to do it this way to avoid cluttering the controller nodes with one more install.